CanvasColorSpaceProposal.md

```diff
@@ -198,9 +198,134 @@ The ``getImageData`` method is responsible for converting the data from the canv
 ## Examples
 
 ### Selecting the best color space match for the user agent's display device
-<pre>
-var colorSpace = window.matchMedia("(color-gamut: p3)").matches ? "display-p3" : "srgb";
-</pre>
+
+This example selects a wide color gamut canvas only if the underlying display has a P3 gamut or larger.
+
+```html
+  // Note that the gamut is named p3, but the color space is 'display-p3'.
+  var matchingColorSpace = window.matchMedia(
+      '(color-gamut: p3)').matches ? 'display-p3' : 'srgb';
+  var canvas = document.getElementById('Canvas');
```

Comment:
Maybe call the canvas elements throughout "myCanvas" or similar, just to avoid confusion? Wasn't clear to me why this was called Canvas and the one in the next example called DefaultCanvas.

Reply:
I changd the element ID to be 'MyCanvas' throughout.

Reply:
Looks generally good. There are a few different canvas names but no big deal.